packages/mapviewer/src/modules/i18n/locales/en.json

```diff
@@ -513,12 +513,16 @@
     "print_scale": "Scale",
     "print_underway_message": "We ask you for your patience. A multi-page print is underway, which can take some time.",
     "problem_announcement": "Report problem",
+    "profile_could_not_generate": "Error : the profile could not be generated",
```

Comment:
The whitespace before ':' can (should?) be avoided I guess? Same in French, but there I don't know if it's maybe required 😀